src/plugins/discover/public/ui_metric/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const DATASOURCE_METRIC_SUFFIX = 'datasource_queries_count';
```

Comment:
we could probably put the non discover specific constants in the data plugin common so it can be reusable

Reply:
@kavilla I was thinking of slightly different approach where each plugin can have a folder called ui_metric which will contain all the information related to ui_metric specific to that plugin over there. So each plugin can independently maintain all the constants and events that they want to be tracked in one place.